File: 1차원 배열/Average.c
#define _CRT_SECURE_NO_WARNINGS 
#include <stdio.h>
int main() {
    
    int num, i;
    double arr[1000], max = 0, min = 0;
    scanf("%d", &num);
    for (i = 0; i < num; i++) {
        scanf("%lf", &arr[i]);

        if (max < arr[i])
            max = arr[i];
    }

    for (i = 0; i < num; i++) {
        arr[i] = arr[i] / max * 100.0;
        min += arr[i];
    }

    printf("%.2lf", (min / (double)num));
}

File: C 언어/JAVA/src/Average.java
import java.util.Scanner;

public class Average {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int num, i;
        double[] arr = new double[1000];
        double max = 0, min = 0;

        num = sc.nextInt();

        for (i = 0; i < num; i++) {
            arr[i] = sc.nextInt();

            if (max < arr[i])
                max = arr[i];
        }
        for (i = 0; i < num; i++) {
            arr[i] = arr[i] / max * 100;
            min += arr[i];
        }
        double s_num = (double) num;
        System.out.println(min / num);
    }

}


